backend/src/main/backend/Misc.txt

```diff
@@ -0,0 +1,32 @@
+Problems:
+--> [FIXED?]
+    the table class is too general. each instance of the class can create
+    tables for all the other tables.
+    For example, the Table object with name "tblMessages" can be used to 
+    create a new table in the database by a different name, "tblLikes."
+    * Need to make Table class more specific
+    * Store a list of Tables in Database.java?
+
+--> [IN PROGRESS]
+    App.java is not yet fully updated to be compatible with the new
+    Database.java
+    * each call to insert a row to a table will need to send the new row info
+      in a list. the order of this list will be sensitive (see insertRow())
+
+
+
+Other Notes:
+--> The Table object is different from a table created in the database. each
+    table in the database has a corresponding Table object that allows us to
+    perform operations on the table using generalized prepared statements.
+
+--> Some methods and operations have not yet been standardized:
+    * The _select_ methods all have different return types based on other.
+      classes this is making it more difficult to standardize them
+
+--> Originally, the rows in each table had id's and those id variables were
+    according to the table they were in. For example, tblMessages contained
+    a column called message_id.
+    These were all changed to just row_id to allow for generality in Table.java
```

Comment:
Small naming thing that applies in a few places - I know in school they sometimes recommend using Hungarian Notation, but that's typically only seen in a select few languages where typing isn't present.
`row_id` and `message_id` can just be simplified to `id` as long as you have the container context present - as in, the `id` field in the `message` table is the row id for that message. Similarly, `mDropTable` could be replaced with `dropTable`. And database tables don't need the `tbl` prefix as they're being referenced as tables anyway.